dev/conductor/lib/next.dart

```diff
@@ -106,10 +106,10 @@ void runNext({
         previousCheckoutLocation: state.engine.checkoutPath,
       );
       // check if the candidate branch is enabled in .ci.yaml
-      if (!engine.ciYaml.enabledBranches.contains(state.engine.candidateBranch)) {
-        engine.ciYaml.enableBranch(state.engine.candidateBranch);
+      if (!(await engine.ciYaml).enabledBranches.contains(state.engine.candidateBranch)) {
```

Comment:
You should store this as a separate variable then reference it here in this block

```
final engineCiYaml = await engine.ciYaml
if (!engineCiYaml.enabledBranches.contains...)
```

Reply:
The one-liner approach was adopted throughout this whole PR. The reason is because there was a bug I caught in `roll_dev.dart`. I refactored   ` if (skipTagging && !repository.isCommitTagged(commit)) {`.  into 
```final bool isCommitTagged = await repository.isCommitTagged(commit);
  if (skipTagging && !isCommitTagged) {
```
But that was wrong, since `repository.isCommitTagged` runs some git commands, and the `skipTagging` is supposed to short-circuit those commands. With my refactor,  the desired `skipTagging` short-circuit no longer takes place. Hence, I refactored to the one-liner await `if (skipTagging && !(await repository.isCommitTagged(commit))) {`. 

As a precaution, I used the one-liner solution wherever I can throughout the code to avoid this situation from happening.

Reply:
In this block we're not short circuiting, so in preference of more readable code lets split the await out